### PR TITLE
Adding the modulo implementation [Issue 1]

### PR DESCRIPTION
**Pull request by tomichaw**
#1

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+all: modulo_lib.o
+
+modulo_lib.o: modulo_lib.cc modulo_lib.h 
+	g++ -c modulo_lib.cc -o $@
```
Comment (birukw) [inline Makefile]: You can use the make automatic variable $< to reference the first prerequisite (modulo_lib.cc in this case).

File: modulo_lib.cc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#include "modulo_lib.h"
+#include <iostream>
+
+int modulo_multiply(int a, int b, int mod) {
+	return (a * b) % (mod + 1);
```
Comment (birukw) [inline modulo_lib.cc:5]: Are you sure this is the correct implementation of modulo_multiply? I don't think you need the +1.

File: Makefile (hunk 1 above)
Comment (tomichaw) [reply Makefile]: Wow, I didn't know that. Thanks! Done.

File: modulo_lib.cc (hunk 2 above)
Comment (tomichaw) [reply modulo_lib.cc:5]: I read on some website that this is the correct implementation.

Comment (birukw) [reply modulo_lib.cc:5]: Ok then, let's see what happens when you run unit tests. Please make sure your unit tests handle all corner cases.

**Review by birukw — APPROVED** [review]
LGTM. Thanks for doing this so quickly!
